eva-pipeline/src/main/java/embl/ebi/variation/eva/VariantJobsArgs.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2015 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package embl.ebi.variation.eva;
+
+import embl.ebi.variation.eva.pipeline.steps.*;
+import org.opencb.biodata.models.variant.VariantSource;
+import org.opencb.biodata.models.variant.VariantStudy;
+import org.opencb.datastore.core.ObjectMap;
+import org.opencb.opencga.lib.common.Config;
+import org.opencb.opencga.storage.core.variant.VariantStorageManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ *
+ * Class to extract configuration from properties files and from command line.
+ * Default values are in resources/application.properties
+ *
+ * @author Diego Poggioli &lt;diego@ebi.ac.uk&gt;
+ *
+ * TODO: 20/05/2016 add type validator?
+ */
+@Component
+public class VariantJobsArgs {
+    private static final Logger logger = LoggerFactory.getLogger(VariantJobsArgs.class);
+
+    ////common
+    @Value("${input:}") private String input;
```

Comment:
Will this provide a default "", preventing the pipeline to break if no input is provided? It should break.